.github/workflows/reusable_gpu.yml

```diff
@@ -100,6 +100,7 @@ jobs:
           -DUMF_BUILD_${{inputs.name}}_PROVIDER=ON
           -DUMF_TESTS_FAIL_ON_SKIP=ON
           ${{ matrix.os == 'Ubuntu' && matrix.build_type == 'Debug' && '-DUMF_USE_COVERAGE=ON' || '' }}
+          ${{ inputs.name == 'LEVEL_ZERO' && '-DCMAKE_SUPPRESS_REGENERATION=ON' || '' }}
```

Comment:
this is not level-zero specific so please remove inputs.name == 'LEVEL_ZERO'
also please add a comment "this is a WA for concurrent builds blabla"